src/main/proxy.ts

```diff
@@ -283,3 +283,16 @@ export const getProxyCertificateContent = () => {
   }
   return undefined
 }
+
+export async function getProxyArguments(
+  settings: ProxySettings,
+  prefix = '--'
+): Promise<string[]> {
+  const spki = await getCertificateSPKI()
+  const port = settings.port
+
+  return [
+    `${prefix}proxy-server=http://localhost:${port}`,
+    `${prefix}ignore-certificate-errors-spki-list=${spki}`,
+  ]
+}
```

Comment:
I was not sure where to put this function. It ended up in `proxy.ts` because I used the `getCertificateSPKI` but it feels more related to browsers.